src/atlas/interpolation/method/sphericalvector/SphericalVector.cc

```diff
@@ -0,0 +1,296 @@
+/*
+ * (C) Crown Copyright 2023 Met Office
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include "atlas/library/defines.h"
+
+#include <cmath>
+#include <tuple>
+#include <utility>
+
+#include "atlas/array/ArrayView.h"
+#include "atlas/array/helpers/ArrayForEach.h"
+#include "atlas/array/Range.h"
+#include "atlas/field/Field.h"
+#include "atlas/field/FieldSet.h"
+#include "atlas/interpolation/Cache.h"
+#include "atlas/interpolation/Interpolation.h"
+#include "atlas/interpolation/method/MethodFactory.h"
+#include "atlas/interpolation/method/sphericalvector/SphericalVector.h"
```

Comment:
Normally the header specific to the class implementation comes before any other include, so to guarantee the implementation sees the same order of includes from the declarion

Reply:
Done.